doc/operator-application-capabilities.md

```diff
@@ -629,6 +636,116 @@ oc create secret generic threescale-provider-account --from-literal=adminURL=htt
 
 The operator will gather required credentials automatically for the default 3scale tenant (provider account) if 3scale installation is found in the same namespace as the custom resource.
 
+## OpenAPI custom resource
+
+### Features
+
+* [OpenAPI 3.0.2](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md) specification
+  * Just a single top level security requirement supported. Operation level security requirements not supported.
+  * Only first `server.url` element in servers list parsed as private url. Path and operation level `servers` elements not supported.
+  * Supported security schemes: `apiKey`.
+* OpenAPI spec document can be read from:
+  * ConfigMap
```

Comment:
I was wondering...

Can the OpenAPI spec contain some kind of sensitive information? If so, we should use a Secret instead of a ConfigMap. If not, a ConfigMap is already ok.

Reply:
I thought about that too.

For the OpenID auth, 3scale required issuer data which contains some secret token. For the toolbox, the customers requested this issuer to be provided in the command line option instead of the OpenAPI spec, to keep OpenAPI public. So, I guess the customers want to keep OpenAPI spec public without any sensitive data. I do not know if this is the case for all cases. 

I will update to secret, just in case.